test/cassettes/test_logon.yml

```diff
@@ -25,12 +25,12 @@ http_interactions:
       Server:
       - Microsoft-HTTPAPI/2.0
       Date:
-      - Wed, 09 Jan 2019 22:58:07 GMT
+      - Mon, 14 Jan 2019 22:56:14 GMT
       Content-<syspro_company>ength:
-      - '36'
+      - '32'
     body:
       encoding: UTF-8
-      string: '8513E1BCC242674D9F3CC6CD27658BFA00  '
+      string: 'ERROR: Invalid operator password'
```

Comment:
Should this cassette being failing the login?

Reply:
no...